associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl

```diff
@@ -354,120 +486,212 @@
             </group>
           </if>
           <else>
-            <text variable="edition" suffix=" ed."/>
+            <text variable="edition"/>
```

Comment:
you'll want a group around this so the term doesn't get written if there's no edition (exactly like you have above)

Reply:
Understood. Thanks!